mne/forward/forward.py

```diff
@@ -410,6 +410,12 @@ def read_forward_solution(fname, force_fixed=None, surf_ori=None,
     See Also
     --------
     write_forward_solution, make_forward_solution
+
+    .. note:: Forward solutions with free orientation are always stored on disk
+              in X/Y/Z RAS coordinates. To obtain surface-oriented forward
+              operators after reading the forward solution with
+              read_forward_solution, please apply convert_forward_solution
```

Comment:
```
:func:`convert_forward_solution`
```
same with `read_forward_solution`

These get rendered as docs online and it's nice if they link properly